### PR TITLE
Add ML collect data script

### PR DESCRIPTION
**Pull request by erezh31**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import demistomock as demisto
```
Comment (meirwah) [inline Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py:1]: why you need this?

File: Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+comment: Collect machine learning data from the environment
```
Comment (meirwah) [inline Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.yml:1]: Collect telemetry is better here to say I think

File: Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py (hunk 1 above)
Comment (erezh31) [reply Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py:1]: It's in every content script, this is the template (it's actually removed when the automation deployed)

**Comment by erezh31** [comment]
@meirwah fixed comments

File: Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.yml [hunk 3]
```diff
@@ -11,4 +11,4 @@ tags:
   - ml
 timeout: '0'
 type: python
-deprecated: true
\ No newline at end of file
+deprecated: true
```
Comment (meirwah) [inline Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.yml:14]: why is this?

Comment (erezh31) [reply Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.yml:14]: we put this to all internal scripts that the user should not run.. like the auto mapper or sane doc report

**Pull request merged**